File: Bars.GkhGJI/Entities/ProtocolMhc/ProtocolMhc.cs
namespace Bars.GkhGji.Entities
{
    using System;

    using Gkh.Entities;

    /// <summary>
    /// Протокол органа муниципального жилищного контроля
    /// </summary>
    public class ProtocolMhc : DocumentGji
    {
        /// <summary>
        /// Тип исполнителя документа
        /// </summary>
        public virtual ExecutantDocGji Executant { get; set; }

        /// <summary>
        /// Контрагент
        /// </summary>
        public virtual Contragent Contragent { get; set; }

        /// <summary>
        /// Физическое лицо
        /// </summary>
        public virtual string PhysicalPerson { get; set; }

        /// <summary>
        /// Реквизиты физ. лица
        /// </summary>
        public virtual string PhysicalPersonInfo { get; set; }

        /// <summary>
        /// Муниципальное образование (Орган прокуратуры, вынесший постановление)
        /// </summary>
        public virtual Municipality Municipality { get; set; }

        /// <summary>
        /// Дата поступления в ГЖИ
        /// </summary>
        public virtual DateTime? DateSupply { get; set; }
    }
}

File: Bars.GkhGJI/libs/B4/model/ProtocolMhc.js
Ext.define('B4.model.ProtocolMhc', {
    extend: 'B4.model.DocumentGji',
    idProperty: 'Id',
    proxy: {
        type: 'b4proxy',
        controllerName: 'ProtocolMhc'
    },
    fields: [
        { name: 'Inspection', defaultValue: null },
        { name: 'InspectionId' },
        { name: 'Executant', defaultValue: null },
        { name: 'Contragent', defaultValue: null },
        { name: 'Municipality', defaultValue: null },
        { name: 'State', defaultValue: null },
        { name: 'DateSupply' },
        { name: 'PhysicalPerson' },
        { name: 'PhysicalPersonInfo' },
        { name: 'TypeDocumentGji', defaultValue: 130 }
    ]
});